.github/workflows/python.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
Python 3.12 is also a [supported Lambda runtime](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).

Reply:
Hi @mrickard,
Yes, even though Python 3.12 is a supported runtime for lambda, the support has not yet been released by serverless in v3. Python 3.12 support has been recently added to the https://github.com/serverless/serverless/issues/12325. The release has not made it to v3 but v4, which is in [beta](https://github.com/serverless/serverless/tree/v4.0).